src/test/testUtil.ts

```diff
@@ -68,6 +68,13 @@ export async function createTestWorkspaceFolder(name?: string): Promise<vscode.W
     }
 }
 
+export async function createTestFile(fileName: string): Promise<vscode.Uri> {
+    const tempFolder = await makeTemporaryToolkitFolder()
```

Comment:
this will let the test teardown delete the temp dir:

    testTempDirs.push(tempFolder)